File: notification/src/main/java/com/ezbuy/notification/repository/query/NotificationQuery.java
package com.ezbuy.notification.repository.query;

public interface NotificationQuery {
    String insertNotification = "INSERT INTO notification (id, sender, severity, notification_content_id, content_type, create_at, create_by, update_at, update_by, category_id, status, expect_send_time)\n" +
            "VALUES(:id, :sender, :severity, :notificationContentId, :contentType, :createAt, :createBy, :updateAt, :updateBy, :categoryId, :status, :expectSendTime);";
}

File: notification-service/src/main/kotlin/com/ezbuy/notificationservice/repository/query/NotificationQuery.kt
package com.ezbuy.notificationservice.repository.query

interface NotificationQuery {
    companion object {
        const val INSERT_NOTIFICATION = """
            INSERT INTO notification (id, sender, severity, notification_content_id, content_type, create_at, create_by, update_at, update_by, category_id, status, expect_send_time)
            VALUES (:id, :sender, :severity, :notificationContentId, :contentType, :createAt, :createBy, :updateAt, :updateBy, :categoryId, :status, :expectSendTime);
        """
    }
}
